paasta_tools/check_spark_jobs.py

```diff
@@ -34,6 +39,11 @@ def parse_args():
         action='store_true',
         help='Skip notifying the teams that own each framework',
     )
+    parser.add_argument(
+        '--email-domain',
+        default=None,
```

Comment:
I think you could get away with `socket.getfqdn().split('.', 1)[1]`

Reply:
`dev.yelpcorp...`? I tried this, Gmail did not like it.